### ISSUE TITLE
Editors on page stuck at 5px height and width

### ISSUE THREAD

**Issue by CBlaize** [egoist/vue-monaco#44]
Hi, thanks for the great vue component!.

I am trying to render multiple editors on the same page.  However, when I load the page, all of the editors show up as a 5px black square.  It's as if the editor is not yet finished loading before it is rendered.

If I add a v-if on the editors and toggle the v-if value, then it appears as expected.

Any thoughts as to why this may be happening?  Spent awhile debugging it to no avail yet.

**Comment by MyCodeGame** [comment]
set automaticLayout=true

**Comment by dzcpy** [comment]
@MyCodeGame Can you please elaborate how to set it? I tried `<MonacoEditor :options="{ automaticLayout: true }">` but it doesn't seem to work

**Comment by WhatFreshHellIsThis** [comment]
Yes, this is driving me crazy, it's very hard to resolve for some reason and creeps back up just when I think it's solved.  This really needs to be addressed, it should just take up all the available space automatically.

**Comment by balhelhas** [comment]
Hello, so the issue comes from version 1.1.0 to 1.2.0. As a workaround till this is fixed revert to 1.1.0.
